lang/cs/Org.Apache.REEF.Tang/Annotations/NamedParameter.cs

```diff
@@ -30,9 +31,11 @@ public class NamedParameterAttribute : System.Attribute
         public Type DefaultClass { get; set; }
         public string[] DefaultValues { get; set; }
         public Type[] DefaultClasses { get; set; }
+        public string Alias { get; set; }
```

Comment:
Do these new attributes need to be `public`? I know that the rest of them are public for some reason, but if can make these `internal`, we wouldn't increase our `public` footprint.